<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAdministrateur</class>
 <widget class="QMainWindow" name="UIAdministrateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>961</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabProfile">
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>40</y>
        <width>961</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIdProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Id :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditIdProfile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLoginProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Login :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditLoginProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelFirstNameProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>First name :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditFirstNameProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelPasswordProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Password :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditPasswordProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelConfirmPasswordProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Confirm password :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEditConfirmPasswordProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditLastNameProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelLastNameProfile">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Last name :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>961</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonModifyProfile">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabUsers">
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>951</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConfirm">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonList">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReset">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>40</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>961</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelFindBy">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Find user by</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFindBy">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Username</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSearch">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelId_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Id :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditId_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLogin">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Login :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditLogin">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelFirstName">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>First name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditFirstName">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPassword_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditPassword_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelConfirmPassword_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirm password :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditConfirmPassword_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelRole_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Role</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="comboBoxRole_3">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>ADMINISTRATEUR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FORMATEUR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ETUDIANT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RESPONSABLE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelLastName">
         <property name="text">
          <string>Last name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditLastName"/>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>951</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonCreate">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonModify">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="labelConnectedUserTitle">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Connected user :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelConnectedUser">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;dynamic&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonExit_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>470</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Tahoma</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
